src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubAppCredentials.java

```diff
@@ -124,23 +158,56 @@ public Secret getPrivateKey() {
         return privateKey;
     }
 
-    /**
-     * Owner of this installation, i.e. a user or organisation, used to differentiate app installations
-     * when the app is installed to multiple organisations / users.
-     *
-     * <p>If this is null then call listInstallations and if there's only one in the list then use
-     * that installation.
-     *
-     * @return the owner of the organisation or null.
-     */
+    /** @deprecated Use {@link #getRepositoryAccessStrategy}. */
+    @Deprecated
     @CheckForNull
     public String getOwner() {
-        return owner;
+        return null;
     }
 
-    @DataBoundSetter
+    // This method is not deprecated or restricted only to preserve compatibility with existing CasC YAML files.
+    /** Do not call this method, use {@link #setRepositoryAccessStrategy} instead. */
```

Comment:
Alternatively, we could mark this method `@Deprecated`, in which case all CasC users that configure GitHub App credentials would have to edit their YAML scripts when applying this plugin update to avoid startup failures. I do not know what approach is generally preferred when making significant configuration changes, so I went for trying to preserve CasC compatibility for now.